tests/Mocks/StorageEngineMock.cpp

```diff
@@ -948,7 +948,7 @@ bool StorageEngineMock::inRecoveryResult = false;
 StorageEngineMock::StorageEngineMock(arangodb::application_features::ApplicationServer& server)
     : StorageEngine(server, "Mock", "",
                     std::unique_ptr<arangodb::IndexFactory>(new IndexFactoryMock())),
-      _releasedTick(0) {
+      _releasedTick(0), vocbaseCount(0){
```

Comment:
```suggestion
      vocbaseCount(0), _releasedTick(0){
```
That will fix error on Mac